### PR TITLE
:tada: first version of tartiflette.io website

### PR DESCRIPTION
**Pull request by tsunammis**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.DS_Store
```
Comment (achedeuzot) [inline .gitignore]: This should be in your global system wide gitignore ;)

File: website/README.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
+This website was created with [Docusaurus](https://docusaurus.io/).
```
Comment (achedeuzot) [inline website/README.md]: Why is the sample site is in the project ? :P

File: website/static/img/medium.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="195px" height="195px" viewBox="0 0 195 195" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
+    <title>Monogram</title>
```
Comment (achedeuzot) [inline website/static/img/medium.svg]: title should be medium :P

File: website/static/img/slack.svg [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="291px" height="278px" viewBox="0 0 291 278" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
+    <title>Artboard</title>
```
Comment (achedeuzot) [inline website/static/img/slack.svg]: title should be slack

File: website/static/img/twitter.svg [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="291px" height="278px" viewBox="0 0 291 278" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
+    <title>Artboard</title>
```
Comment (achedeuzot) [inline website/static/img/twitter.svg]: should be twitter

File: .gitignore (hunk 1 above)
Comment (tsunammis) [reply .gitignore]: 👍 It was in the default `.gitignore` file of docusaurus

File: website/package.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (abusi) [inline website/package.json]: this looks like an "example" file ?

File: website/static/css/custom.css [hunk 7]
```diff
@@ -0,0 +1,61 @@
+/* your custom css */
```
Comment (abusi) [inline website/static/css/custom.css]: maybe remove this ?

File: website/static/img/medium.svg (hunk 3 above)
Comment (tsunammis) [reply website/static/img/medium.svg]: fixed 👍

File: website/static/img/slack.svg (hunk 4 above)
Comment (tsunammis) [reply website/static/img/slack.svg]: 👍

File: website/static/img/twitter.svg (hunk 5 above)
Comment (tsunammis) [reply website/static/img/twitter.svg]: 👍

File: website/static/css/custom.css (hunk 7 above)
Comment (tsunammis) [reply website/static/css/custom.css]: 👍

File: website/package.json (hunk 6 above)
Comment (tsunammis) [reply website/package.json]: This is a relevant file

**Comment by tsunammis** [comment]
@abusi fixed

**Pull request merged**